client/src/app/root-store/huggingface-model-store/huggingface-model-store.state.ts

```diff
@@ -0,0 +1,21 @@
+import { IHFModelsData } from '@core/services/api/rest/huggingface.service';
+
+import { ErrorState } from '@store/state';
+
+export interface State {
+  modelsData: IHFModelsData;
+  isLoading: boolean;
+  error: ErrorState;
+  modelReadme: string;
+  isLoadingReadme: boolean;
+  modelReadmeError: ErrorState;
```

Comment:
How about differentiating loading and error fields, e.g.
1. By naming
```suggestion
  modelsData: IHFModelsData;
  isModelsDataLoading: boolean;
  modelsDataError: ErrorState;
  modelReadme: string;
  isLoadingReadme: boolean;
  modelReadmeError: ErrorState;
```
2. Or by adding one more level for readme
```suggestion
  modelsData: IHFModelsData;
  isLoading: boolean;
  error: ErrorState;
  selectedModelDetails: {
    readme: string;
    isLoading: boolean;
    error: ErrorState;
  },
  modelReadme: string;
  isLoadingReadme: boolean;
  modelReadmeError: ErrorState;
```

Reply:
fixed